Source/Plugins/NetworkEvents/NetworkEvents.cpp

```diff
@@ -85,14 +85,22 @@ void NetworkEvents::restartConnection()
 
 void NetworkEvents::createEventChannels()
 {
-	EventChannel* chan = new EventChannel(EventChannel::TEXT, 1, MAX_MESSAGE_LENGTH, CoreServices::getGlobalSampleRate(), this);
+    juce::int64 timestamp = CoreServices::getGlobalSampleRate();
```

Comment:
This is not a timestamp but a sample rate. The name is confusing but, more importantly, it should be a float, not a int64

Reply:
Oh yep, misread the function. Fixed!